factory.go

```diff
@@ -0,0 +1,58 @@
+package t1k
+
+import (
+	"errors"
+	"net"
+	"time"
+)
+
+// ConnectionFactory 连接工厂
+type ConnectionFactory interface {
+	//生成连接的方法
+	Factory() (interface{}, error)
```

Comment:
在 `factory.go` 中，`ConnectionFactory` 接口的 `Factory()` 方法缺少注释说明。建议为该方法添加注释，说明其功能和返回值。

```suggestion
// Factory 生成连接的方法
Factory() (interface{}, error)